at_client/atclient.py

```diff
@@ -406,20 +417,25 @@ def handle_event(self, queue, at_event):
                         key = event_data["key"]
                         encrypted_value = event_data["value"]
                         ivNonce = event_data["metadata"]["ivNonce"]
+                        self.secondary_connection.execute_command("notify:remove:" + event_data["id"])
                         try:
                             encryption_key_shared_by_other = self.get_encryption_key_shared_by_other(SharedKey.from_string(key=key))
                             decrypted_value = EncryptionUtil.aes_decrypt_from_base64(encrypted_text=encrypted_value.encode(), self_encryption_key=encryption_key_shared_by_other, iv=base64.b64decode(ivNonce))
                             new_event_data = dict(event_data)
                             new_event_data["decryptedValue"] = decrypted_value
                             new_at_event = AtEvent(AtEventType.DECRYPTED_UPDATE_NOTIFICATION, new_event_data)
-                            queue.put(new_at_event)
+                            #What should I do about legacy code that is expecting the decrypted event to be in the queue?
+                            self.decrypted_events.put(new_at_event)
+                            #self.queue.put(new_at_event)
                         except Exception as e:
                             print(str(time.time()) + ": caught exception " + str(e) + " while decrypting received data with key name [" + key + "]")
             except Empty:
                 pass
         else:
             raise Exception("You must assign a Queue object to the queue paremeter of AtClient class")
         
+    #I  want to leave this here instead of the AtNotificationService 
+    # because it's logic is fairly related to the AtClient instead of the NotficationService (despite the name..)  
     def notify(self, at_key : AtKey, value, operation = OperationEnum.UPDATE, session_id = str(uuid.uuid4())):
```

Comment:
#I  want to leave this here instead of the AtNotificationService 
    # because it's logic is fairly related to the AtClient instead of the NotficationService (despite the name..)